docs/application/dotnet/guides/account/app-history.md

```diff
@@ -84,24 +84,24 @@ To retrieve application usage statistics for a given time period, and check deta
 
 To retrieve battery usage statistics for a given time period, and check detailed statistics information, such as the battery consumption of the used applications:
 
-1.  To retrieve the battery consumption per application, create an instance of the [Tizen.Context.AppHistory.BatteryStatistics](https://samsung.github.io/TizenFX/latest/api/Tizen.Context.AppHistory.BatteryStatistics.html) class:
+1.  To retrieve the battery consumption per application, create an instance of the [Tizen.Context.AppHistory.BatteryStatistics](/application/dotnet/api/TizenFX/latest/api/Tizen.Context.AppHistory.BatteryStatistics.html) class:
 
-    ```
+    ```csharp
     BatteryStatistics batteryConsumedApp = new BatteryStatistics(BatteryStatistics.SortOrderType.ConsumptionMost);
     ```
 
 2.  To get the information about the application battery consumption, use the `Query()` method of the `Tizen.Context.AppHistory.BatteryStatistics` class.
 
     For example, to retrieve battery consumption history since the device was last fully charged, use a `DateTime` instance returned by the `GetLastFullyChargedTime()` method as the `startTime` parameter of the `Query()` method:
```

Comment:
```suggestion
    For example, to retrieve battery consumption history since the device was last fully charged, use a `DateTime` instance returned by `GetLastFullyChargedTime()` as the `startTime` parameter of  `Query()`:

Reply:
done